lib/settings/pages/settings_page.dart

```diff
@@ -45,6 +51,101 @@ class SettingsPage extends StatelessWidget {
             toolbarHeight: 70.0,
             pinned: true,
           ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 18),
+              child: SearchAnchor(
+                viewHintText: l10n.search,
+                builder: (BuildContext context, SearchController controller) {
+                  if (Platform.isIOS) {
```

Comment:
Hmm, I think maybe we can leave this as the Material `SearchBar` if that's all good with you! Or remove it completely if we use `SearchAnchor.bar`.

My main reasoning for this is that we use `SearchAnchor` for suggestions and results, which is more of a "Material" component, rather than a component that uses Apple's HIG guidelines. Because of this, it might feel weird that we're switching between two different styles (Apple's HIG and Google's Material).

This is also consistent with the search page, where we use Material's Search UI!